packages/flutter_tools/lib/src/android/gradle_errors.dart

```diff
@@ -50,8 +50,8 @@ enum GradleBuildStatus {
   retryWithAarPlugins,
 }
 
-/// Returns a simple test function that evaluates to [true] if
-/// [errorMessage] is contained in the error message.
+/// Returns a simple test function that evaluates to `true` if at least one of
+/// `errorMessages` is contained in the error message.
```

Comment:
are we migrating to backticks?

Reply:
No - but brackets mean dartdoc will try to link these to a class or class member. We don't use them for parameters or langauge keywords.

Reply:
(I make this mistake all the time - now in the framework it gets caught but I don't think we check dartdoc validity in the tool)

Reply:
ah - I see